source/adapter.screenshare.js

```diff
@@ -75,9 +74,9 @@
     };
 
   } else if (window.navigator.webkitGetUserMedia) {
-    baseGetUserMedia = window.navigator.getUserMedia;
 
     navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      baseGetUserMedia = window.navigator.getUserMedia;
```

Comment:
@johache I have a proposal as well, since I never liked the idea of overwriting the `getUserMedia()` API to do handle the cross-browser screensharing polyfill. I propose we use a different method name called `getScreenMedia()`.

```
navigator.getScreenMedia = function (constraints, ..) {
  // Modifications here
  navigator.getUserMedia(constraints, ..);
}; 

navigator.mediaDevices.getScreenMedia = function (constraints, ..) {
  // Modifications here
  navigator.mediaDevices.getUserMedia(constraints, ..);
}; 

window.getScreenMedia = navigator.getScreenMedia;
```